modules/post/multi/recon/local_exploit_suggester.rb

```diff
@@ -118,7 +120,11 @@ def run
     results = []
     @local_exploits.each do |m|
       begin
-        checkcode = m.check
+        begin
+          checkcode = m.check
+        rescue => e
+          elog("#{m.shortname} failed to run", error: e)
```

Comment:
Let's be consistent on providing verbose output and not just send the error to a log.
```suggestion
          elog("#{m.shortname} failed to run", error: e)
          vprint_error "#{e.class} #{m.shortname} failed to run: #{e.message}"
```

Reply:
Logging happens later:
```
        if checkcode.nil?
          vprint_error "#{m.fullname}: Check failed"
          next
        end
```

But this could definitely be tweaked to be more clear to the user that something went more wrong than normal 😄